Cookbook/Technical-Documents/WhatIDontKnow.md

```diff
@@ -0,0 +1,183 @@
+# What I don't know, but should
+
+## Congratulations! You've joined Babylon...
+Welcome to the amazing iOS team! 
+Babylon is a fast-growing company. Currently, we are on-boarding ~30 new-starters weekly. Many of them are engineers like yourself. Our team is scattered across three offices: London West, London East and US office. On top of that, we have fully remote teammates. As you can imagine this poses almost biblical challenges, but don’t worry :)
+
+## Like others before you...
+The first week can be overwhelming. New code-base, emails and non-stop messages on Slack can make for a mind-numbing first experience. We don’t promise the information fire hose stops after the first day, but just hear what people who went through before you said:
+
+Q: What did you like during your first week?
+> “1-on-1 with Rui and squad PMs, meet team, talk a lot within the squad, pair with a couple of people on iOS team. Gone out to drinks with the squad :)”
```

Comment:
Since Rui is no longer here, this could be confusing. 
```suggestion
> “1-on-1 with Rui _(line manager)_ and squad PMs, meet team, talk a lot within the squad, pair with a couple of people on iOS team. Gone out to drinks with the squad :)”
```

Reply:
I am always here. 😄